accounting/acct.proto

```diff
@@ -0,0 +1,147 @@
+// Accounting for gnXi operations.
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// A protobuffer definition used for logging events reported to gNxI server.
+// This file assumes the following nomenclature:
+//
+//   Device >---connects---> gNxI accounting Server
```

Comment:
Is this right? Are we expecting that this is a dial-out service? I suggest that it's not given the model that we have today -- this does mean that accounting might be lost if you don't have a connection though. So we should likely think about what we want to do in this regard (this is the same concern as any telemetry).

Do we expect that there is also a local accounting log? We probably could say something like this, and have this included in a model for the configuration.